### ISSUE TITLE
Bug in testing instrument docstrings

### ISSUE THREAD

**Issue by aburrell** [pysat/pysat#339]
**Describe the bug**
When reading the docstrings in several of the testing instruments (specifically the xarray ones), `tag` is flagged as the keyword argument to specify the number of times to load.  The actual keyword argument that does this is `sat_id`

**To Reproduce**

```
import pysat
inst = pysat.Instrument(platform=str('pysat'), name=str('testing'),sat_id='6', clean_level='clean')
model = pysat.Instrument(platform=str('pysat'), name=str('testing_xarray'), tag='6', clean_level='clean')

inst.load(yr=2009,doy=1)
model.load(yr=2009,doy=1)
print(inst.index.shape, model.index.shape)
```
((6,), (86400,))
```
model = pysat.Instrument(platform=str('pysat'), name=str('testing_xarray'), sat_id='6', clean_level='clean')
model.load(yr=2009,doy=1)
print(inst.index.shape, model.index.shape)
```
((6,), (6,))

**Desktop (please complete the following information):**
 - OS:  OS X mojave
 - Version 2.7

**Comment by jklenzing** [comment]
Good catch.  Must have missed this in the previous pull when this was updated.

**Comment by aburrell** [comment]
By "good catch" I think you mean "I am sorry you now have a headache from banging your head against the computer" ;-)

**Comment by jklenzing** [comment]
Shoot.  Accidentally committed to develop and pushed.  Reverting.

**Comment by jklenzing** [comment]
Ugh.  Why is develop not a protected branch?

**Comment by jklenzing** [comment]
Closed with merge of #340 and #341.
